packages/react-wildcat/src/server.js

```diff
@@ -107,6 +107,18 @@ function start() {
                 }));
             });
 
+            // Allow custom middleware priority over react/jspm/render from below
+            // otherwise server-only routes will receive 404's.
+            if (appServerSettings.middleware && appServerSettings.middleware.length) {
+                appServerSettings.middleware.forEach(function eachCustomAppMiddleware(middlewareConfigFunction) {
+                    if (typeof middlewareConfigFunction === "function" && middlewareConfigFunction.length >= 1) {
```

Comment:
The intent of checking the function's length is to verify it is following the pattern.

either `()=>{}` or `(app) =>` is expected... (generally the later) if someone has a function that has `(app, somethingElse) => {}` that is incorrect.

thoughts?